esx_service/vmdk_ops.py

```diff
@@ -314,15 +314,25 @@ def find_child(vm_name):
 
 #returns error, or None for OK
 def attachVMDK(vmdk_path, vm_name):
-    vm = findVmByName(vm_name)
+    try:
+       vm = findVmByName(vm_name)
```

Comment:
The reason to ignore it is so that we know what it is and get a traceback on crash to help us debug later. What are we catching and converting to an error code otherwise? I didn't see a good way to do this systematically.